util/collections.go

```diff
@@ -22,4 +24,43 @@ func ListContains(needle string, haystack []string) bool {
 	}
 
 	return false
-}
\ No newline at end of file
+}
+
+// Convert a generic list to a list of strings
+func ToStringList(genericList []interface{}) []string {
+	stringList := []string{}
+
+	for _, value := range genericList {
+		stringList = append(stringList, ToString(value))
+	}
+
+	return stringList
+}
+
+// Convert a generic map to a map from string to string
+func ToStringMap(genericMap map[interface{}]interface{}) map[string]string {
+	stringMap := map[string]string{}
+
+	for key, value := range genericMap {
+		stringMap[ToString(key)] = ToString(value)
+	}
+
+	return stringMap
+}
+
+// Convert a generic map to a map from string to interface
+func ToStringToGenericMap(genericMap map[interface{}]interface{}) map[string]interface{} {
```

Comment:
Did I mention Go is a shitty language? Writing boilerplate code like this (ba-dum-tish) is going to drive me mad.